src/optionals/http/http.c

```diff
@@ -948,6 +973,19 @@ ObjAbstract *newHttpClient(DictuVM *vm, ObjDict *opts) {
                     curl_easy_setopt(httpClient->curl, CURLOPT_SSL_VERIFYHOST, 0);
                     curl_easy_setopt(httpClient->curl, CURLOPT_SSL_VERIFYPEER, 0);
                 }
+            } else if (strstr(key, "follow_redirects")) {
+                if (IS_EMPTY(entry->value)) {
+                    continue;
+                }
+
+                if (!IS_BOOL(entry->value)) {
+                    runtimeError(vm, "HTTP client option \"follow_redirects\" value must be a bool");
+                    return abstract;
```

Comment:
I've just realised an issue here. We aren't handling the failure as we should here, this will probably end up with a segfault in the erroneous scenario.

Instead what we need to do is whenever we raise a runtimeError we return an `EMPTY_VAL`. So `newHttpClient` should look like: `Value newHttpClient(DictuVM *vm, ObjDict *opts)`

And it means `newClient` can just return as:
```
return newHttpClient(vm, opts);
```

Reply:
All set I believe.